src/app/components/footer/footer.component.scss

```diff
@@ -11,8 +11,11 @@
   font-family: Roboto, Helvetica Neue, Arial, sans-serif, Apple Color Emoji,
     Segoe UI Emoji, Segoe UI Symbol;
   text-decoration: none;
+  background-color: transparent;
 
-  border-top: 2px solid #1a191a !important;
+   border-top: 2px solid #1a191a !important;
```

Comment:
Should be removed when it's done we discussed recently

Reply:
i could not do it by padding, since the changes didnt appear on the screen